docs/articles/expensify-classic/spending-insights/Insights.md

```diff
@@ -1,101 +1,90 @@
-- **All Data—Report Level Export**: This report is great if you don't need to see each individual expense but want to see a line-by-line breakdown at a report level (submitter, total amount, report ID, etc.)
-- **Basic Export**: The best way to get a simple breakdown of all your expenses
-- **Canadian Multiple Tax Export**: To gain a better understanding of the various Canadian taxes tied to expenses (GST, PST, etc.)
-- **Category Export**: Use this to see a breakdown of your expenses by Category
-- **Per Diem Export**: Use to export your Per Diem details
-- **Tag Export**: Similar to the Category Export, but for Tags
+## How many expenses can I export in one report?
 
-*These reports will be emailed directly to your email address rather than automatically downloaded.* 
+- Custom reports are best for small-to-medium datasets.
+- For large exports, use a [default export report](https://help.expensify.com/articles/expensify-classic/spending-insights/Default-Export-Templates) from the Reports page.
 
-## How many expenses can I export in one report? 
-The custom export reports are best for small-to-medium chunks of data. If you want to export large amounts of data, we recommend you use a [default export report](https://help.expensify.com/articles/expensify-classic/spending-insights/Default-Export-Templates) that you can run from the Reports page.  
+## What types of custom reports can my Account Manager help create?
 
-## What other kinds of export reports can my Account Manager help me create? 
+We offer various custom reports, including:
 
-We’ve built a huge variety of custom reports for customers, so make sure to reach out to your Account Manager for more details. Some examples of custom reports we’ve built for customers before are: 
-- Accrual Report
+- Accrual Reports
 - Aged Approval Reports
-- Attendee Reporting
-- Audit Report
-- Candidate Spend
-- Category Spend Report
-- Department/Project Spend Report
-- Duplication Report
-- Duty of Care
-- Efficiency 
-- Employee Bank Account Status
-- Employee Details
-- Employee Roles
-- Expense Authorizations by Country
+- Audit Reports
+- Candidate Spend Analysis
 - Expense Reports by Country
-- Expense Reports not posted to finance system
-- Foreign Currency Transaction
-- Fringe Benefit Tax Report
-- HR Report
-- Invoice Billed Transaction Reconciliation
-- Mileage Reports
-- Out of Pocket Expenses for Reimbursement
-- Per Diem Report
-- Reconciliation: Accounting, Bank Statement, Billed Transaction
-- Rejected Report
-- Travel Rule Class
-- Travel Spend
-- Unposted Cash Advance Report
-- Unposted Procurement Aging Report
-- Unposted Travel Aging Report
-- Vendor Spend
-
-{% include faq-end.md %}
+- Employee Bank Account Status
+- Invoice Reconciliation Reports
+- Vendor Spend Analysis
+
+Need something specific? Contact your Account Manager for assistance!
+
+---
```

Comment:
```suggestion
```

Don't need this one at the end if we use the FAQ tag

Reply:
I think the outcome of this comment will be dependent on what we find about the collapsible FAQ section. I'll wait on that.

Reply:
I think even without the FAQ block we don't need this line